forwards_paper.tex

```diff
@@ -225,40 +233,23 @@ \subsection*{Simulation benchmarks}
 coalescent \citep{Neuhauser1997-nn}.  Therefore, these selected mutations must be explicitly tracked in our forward simulation
 and the time savings due to pedigree recording come from not having to record \textit{neutral} mutations.
 
-We implemented our method in \cpp{}, using \fwdpp{} library functions and
-interface code (details below), using a continuum-sites model for both mutation
-and recombination. Simulations were run using \fwdpy{}, a Python package based
-on \fwdpp.
-% $\mu$ and $r$ were the means of Poisson processes, and mutations used the infinite-sites model \citep{Kimura1969-uc}.
-% We varied $N \in (1000, 10000, 50000)$, $\rho \in (10^3, 10^4, 10^5)$.
-We ran all benchmarks on an Ubuntu Linux (version 16.04) system with two 2.6 GHz Intel E5-2650 CPU with
-hyperthreading enabled.
-We ran one simulation at a time and the machine was under minimal load otherwise.
-We used GNU parallel \citep{Tange2011a} to kill any simulation that did not finish within 72 hours,
-and the Linux \texttt{time} command to record run time and peak memory usage of each replicate.
-%% Our simulation scripts separately recorded the time spent in various steps of each run.
-
-%% \krt{currently, we are not doing anything with the detailed timings.  I think that is fine, and that we can delete last
-%%sentence of preceding paragraph.}
-
-
-The total run times for simulations with selection are shown in Figure~\ref{fig:runtimes_selection}.  When tracking
-neutral mutations (instead of the pedigree), run times increase dramatically with increasing region size ($4Nr = 4Nu$ for
-these simulations).  Simulations with $N=5 \times 10^4$ timed out for region sizes larger than $10^3$.
```

Comment:
moved this note about timing out to the figure caption so it doesn't get in the way of the flow